internal/hammer/loadtest/client.go

```diff
@@ -155,42 +155,42 @@ type httpLeafWriter struct {
 	bearerToken string
 }
 
-func (w httpLeafWriter) Write(ctx context.Context, newLeaf []byte) (uint64, error) {
+func (w httpLeafWriter) Write(ctx context.Context, newLeaf []byte) (uint64, uint64, error) {
```

Comment:
If the end goal is that this `loadtest` library is to be reused between at least Tessera and TesseraCT then this return type will need to be revised here. But we can merge this now and do the diffs when this is feature complete to see what we need to do.

Reply:
Correct, the end goal is to use the `loadtest` library from trillian-tessera repo.